course/templates/course/gradebook.html

```diff
@@ -50,7 +50,8 @@
         "scrollX": true,
         "scrollCollapse": true,
         "paging": true,
-        "ordering": false,
```

Comment:
I have no idea why ``"ordering"`` is originally set to be ``false``. It works well if set to be ``true``.